dll/cpl/desk/screensaver.c

```diff
@@ -594,6 +667,40 @@ OnInitDialog(HWND hwndDlg, PDATA pData)
         return FALSE;
     }
 
+    wc.lpfnWndProc = DefWindowProc;
+    wc.hInstance = hApplet;
+    wc.hCursor = NULL;
+    wc.hbrBackground = (HBRUSH)GetStockObject(BLACK_BRUSH);
+    wc.lpszClassName = szPreviewWndClass;
+
+    if (RegisterClass(&wc))
```

Comment:
Better use RegisterClassEx with a WNDCLASSEX.

Reply:
Well [MSDN says](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerclassa) that "You can still use RegisterClass, however, if you do not need to set the class small icon." -- also this is already used for multimon control here: https://git.reactos.org/?p=reactos.git;a=blob;f=dll/cpl/desk/monslctl.c;hb=3aa5e8b897d4962080ecc765ad6b1aa50d6d886a#l1637